doc/DNM-bank.yml

```diff
@@ -34,28 +34,32 @@ paths:
         500:
           $ref: '#/components/responses/500'
 
-    post:
-      summary: Create a new account
+    put:
+      summary: Save or update accounts
       tags:
         - Account
       requestBody:
         required: true
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/AccountInput'
+              type: array
+              items:
+                $ref: '#/components/schemas/AccountInput'
```

Comment:
- Make some checks about the AccountInput and the AccountComponent,
- if the AccountInput has an id (id != null), it is a update, else it is a creation
- Do not forget the overDrafted column
- Match the OpenAPI Component with AccountComponent class in the code
- Same with AccountInput and the Account model